types/statuses.pp

```diff
@@ -0,0 +1,8 @@
+type Servicenow_reporting_integration::Statuses = Array[Enum['failed_changes',
+                                                             'corrective_changes',
+                                                             'intentional_changes',
+                                                             'pending_changes',
+                                                             'all',
+                                                             'none',
+                                                             'unchanged'
```

Comment:
`unchanged` => `no_changes`. Reason is b/c 'pending_changes' reports can also (technically) be unchanged. no_changes better distinguishes reports without any kind of changes, which I feel is a useful distinction to make.

Reply:
Done.